apache/superset
# chore: update docs for new alerts and reporting feature

### SUMMARY
Added docs for setting up and configuring the new alerts and reports in V.1.0

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TEST PLAN
Using the templates outlined in the doc, run these steps to get the new alerts and reporting feature working. After these steps are completed, you should now be able to create and test alerts and reports

1. Create a new directory and create the Dockerfile
2. Build the extended image using the Dockerfile
3. Create the `docker-compose.yaml` file in the same directory
4. Create a new sub directory called `config`
5. Create the `superset_config.py` file in the `config` sub directory
6. Run the image using `docker-compose up` in the same directory as the `docker-compose.py` file

### ADDITIONAL INFORMATION
